### PR TITLE
Richard_Printer

### PR DESCRIPTION
**Pull request by RichardSeverich**

### CODE REVIEW

File: src/test/java/org/fundacionjala/coding/richard/PrinterTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ */
+public class PrinterTest {
+
+    /**
+     *
+     */
+    @Test
+    public void test() {
+        Printer printer = new Printer();
+        System.out.println("printerError Fixed Tests");
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/richard/PrinterTest.java]: Remove this sout

File: src/test/java/org/fundacionjala/coding/richard/PrinterTest.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        String s = "aaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbmmmmmmmmmmmmmmmmmmmxyz";
+        assertEquals("3/56", printer.printerError(s));
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/coding/richard/PrinterTest.java]: Please add one scenario when the string doesn't have errors
for instance
s="aaabbbbhaijjjm"
error_printer(s) => "0/14"

File: src/test/java/org/fundacionjala/coding/richard/PrinterTest.java (hunk 1 above)
Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/coding/richard/PrinterTest.java]: okey.

File: src/test/java/org/fundacionjala/coding/richard/PrinterTest.java (hunk 2 above)
Comment (RichardSeverich) [reply src/test/java/org/fundacionjala/coding/richard/PrinterTest.java]: ok.

**Comment by carledriss** [comment]
LGTM!
Good job! :)

**Pull request merged**